src/k8s/api/v1/path.go

```diff
@@ -0,0 +1,8 @@
+package apiv1
```

Comment:
Can we rename the filename please? Maybe something like `api_version` or just `version`?